{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py

```diff
@@ -16,30 +19,26 @@
 urlpatterns = [
 
     # Admin URLs.
-    url(r'^admin/password_change/$', 'django.contrib.auth.views.password_change',
+    url(r'^admin/password_change/$', auth_views.password_change,
         {'password_change_form': CMSPasswordChangeForm}, name='password_change'),
-    url(r'^admin/password_change/done/$', 'django.contrib.auth.views.password_change_done', name='password_change_done'),
+    url(r'^admin/password_change/done/$', auth_views.password_change_done, name='password_change_done'),
     url(r'^admin/', include(admin.site.urls)),
     url(r'^admin/', include('social_django.urls', namespace='social')),
     {% if cookiecutter.sections == 'no' %}# {% endif %}url(r'^admin/pages/page/sections.js$', sections_js, name='admin_sections_js'),
 
     # Permalink redirection service.
-    url(r'^r/(?P<content_type_id>\d+)-(?P<object_id>[^/]+)/$', 'django.contrib.contenttypes.views.shortcut', name='permalink_redirect'),
+    url(r'^r/(?P<content_type_id>\d+)-(?P<object_id>[^/]+)/$', contenttypes_views.shortcut, name='permalink_redirect'),
 
     # Google sitemap service.
-    url(r'^sitemap.xml$', 'django.contrib.sitemaps.views.index', {'sitemaps': registered_sitemaps}),
-    url(r'^sitemap-(?P<section>.+)\.xml$', 'django.contrib.sitemaps.views.sitemap', {'sitemaps': registered_sitemaps}),
+    url(r'^sitemap.xml$', sitemaps_views.index, {'sitemaps': registered_sitemaps}),
+    url(r'^sitemap-(?P<section>.+)\.xml$', sitemaps_views.sitemap, {'sitemaps': registered_sitemaps}),
 
     # Basic robots.txt.
     url(r'^robots.txt$', TextTemplateView.as_view(template_name='robots.txt')),
 
     # There's no favicon here!
     url(r'^favicon.ico$', generic.RedirectView.as_view(permanent=True)),
-] + static(
-    settings.MEDIA_URL, document_root=settings.MEDIA_ROOT
-) + static(
-    settings.NODE_MODULES_URL, document_root=settings.NODE_MODULES_ROOT
-)
```

Comment:
As far as i can see these have never been used. I know 100% the `node_modules` one hasn't